config/kratos/kratos.yml

```diff
@@ -0,0 +1,84 @@
+version: v0.13.0
+
+dsn: memory
+
+session:
+  lifespan: 720h
+
+serve:
+  public:
+    base_url: http://localhost:4433/
+    cors:
+      enabled: true
+      allow_credentials: true
+      allowed_origins:
+        - http://localhost:3000
+        - http://*.localhost:3000
+      allowed_methods:
+        - POST
+        - GET
+        - PUT
+        - PATCH
+        - DELETE
+      allowed_headers:
+        - Authorization
+        - Cookie
+        - Content-Type
+        - X-Session-Token
+        - X-Csrf-Token
+      exposed_headers:
+        - Content-Type
+        - Set-Cookie
+      debug: true
+  admin:
+    base_url: http://localhost:4434
+
+selfservice:
+  default_browser_return_url: http://localhost:3000
+  allowed_return_urls:
+    - http://localhost:3000
+
+  methods:
+    password:
+      enabled: true
+
+  flows:
+    login:
+      ui_url: http://localhost:3000/auth/login
+
+    registration:
+      ui_url: http://localhost:3000/auth/registration
+
+    settings:
+      ui_url: http://localhost:3000/auth/settings
+
+    recovery:
+      enabled: false
+      
+log:
+  level: debug
+
+secrets:
+  cookie:
+    - PLEASE-CHANGE-ME-I-AM-VERY-INSECURE
+  cipher:
+    - 32-LONG-SECRET-NOT-SECURE-AT-ALL
+
+ciphers:
+  algorithm: xchacha20-poly1305
+
+hashers:
+  algorithm: bcrypt
+  bcrypt:
+    cost: 8
+
+identity:
+  default_schema_id: default
+  schemas:
+    - id: default
+      url: file:///etc/config/kratos/identity.schema.json
+
+courier:
+  smtp:
+    connection_uri: smtps://test:test@mailslurper:1025/?skip_ssl_verify=true
```

Comment:
probably not needed with username only

Reply:
I tried removing it earlier, but I think this is a required property.
```
The configuration contains values or keys which are invalid:
courier.smtp.connection_uri: <nil>
                             ^-- one or more required properties are missing
time=2023-08-11T13:42:24Z level=error msg=Unable to instantiate configuration. audience=application error=map[message:I[#/courier/smtp] S[#/properties/courier/properties/smtp/required] missing properties: "connection_uri" stack_trace:stack trace could not be recovered from error type *jsonschema.ValidationError] service_name=Ory Kratos service_version=v0.13.0
Error: I[#/courier/smtp] S[#/properties/courier/properties/smtp/required] missing properties: "connection_uri"
Usage:
  kratos serve [flags]
Flags:
  -c, --config strings   Path to one or more .json, .yaml, .yml, .toml config files. Values are loaded in the order provided, meaning that the last config file overwrites values from the previous config file.
      --dev              Disables critical security features to make development easier
  -h, --help             help for serve
      --sqa-opt-out      Disable anonymized telemetry reports - for more information please visit https://www.ory.sh/docs/ecosystem/sqa
      --watch-courier    Run the message courier as a background task, to simplify single-instance setup
I[#/courier/smtp] S[#/properties/courier/properties/smtp/required] missing properties: "connection_uri"
```